components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -665,13 +665,11 @@ func prepareIDELaunch(cfg *Config, ideConfig *IDEConfig) *exec.Cmd {
 	if ideConfig.Entrypoint == "/ide/startup.sh" && len(args) == 0 {
 		args = append(args, "{WORKSPACEROOT}")
 		args = append(args, "--port", "{IDEPORT}")
```

Comment:
Can we remove this now or we need to wait some time after deploying this to avoid breaking older workspaces

Reply:
I think we need to make sure that both stable and insiders already running with new config. This PR affects only insiders for now as far as I know.

Why do we need to change this file as a part of this PR?

Reply:
Just doing some minor cleanup as I saw in the logs a warning  that `--hostname` is not a valid server argument